man/systemd.netdev.xml

```diff
@@ -408,6 +408,15 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>VLANProtocol=</varname></term>
+        <listitem>
+          <para>Allows setting the protocol used for VLAN filtering. Takes
+          <option>802.1Q</option> or,
+          <option>802.1AD</option>, and defaults to unset and kernel's default is used.
```

Comment:
we usually use only lower-case enum values in our config files, we should do so here too.